### PR TITLE
log message at per-test level for`perfpipe_pytorch_test_times`

### PR DESCRIPTION
**Pull request by dhuang29**

### CODE REVIEW

File: test/print_test_stats.py [hunk 1]
```diff
@@ -77,25 +79,28 @@ def parse_reports(folder):
...
+
+def build_message(test_case):
     return {
         "normal": {
             "build_pr": os.environ.get("CIRCLE_PR_NUMBER"),
             "build_tag": os.environ.get("CIRCLE_TAG"),
             "build_sha1": os.environ.get("CIRCLE_SHA1"),
             "build_branch": os.environ.get("CIRCLE_BRANCH"),
-            "test_name": test_suite.name,
+            "test_name": test_case.name,
```
Comment (walterddr) [inline test/print_test_stats.py:90]: are we sure `test_case.name == test_suit.name`? 
you can verify this by tailing the scribe category and filter our your messages.

if they dont match, you probably need to tag all 3
```
"test_name": test_suite.name,
"test_case_name": test_case.name,
"test_case_class": test_case.class_name,
```

**Review by walterddr — COMMENTED** [review]
looks good overall. thank you for the contribution. I left a comment on the format. please verify it in scribe/scuba first before reacting to the comment. thanks --Rong

File: test/print_test_stats.py [hunk 2]
```diff
@@ -46,11 +47,12 @@ def print_report(self):
         sorted_tests = sorted(self.test_cases, key=lambda x: x.time)
         test_count = len(sorted_tests)
         print(f"class {self.name}:")
-        print(f"    tests: {test_count} failed: {self.failed_count} skipped: {self.skipped_count} errored: {self.errored_count}")
+        print(
```
Comment (walterddr) [inline test/print_test_stats.py:50]: can you revert this and all other format changes done automatically by the IDE. Seems like the report file
`test-reports` is also involved in the codecov CI job so there might've been some formatting issues here and there getting triggered.

Comment (dhuang29) [reply test/print_test_stats.py]: Thanks for pointing out! Changed it back.

**Comment by dhuang29** [comment]
@walterddr  @malfet Linter(clang-tidy) failed. I rebased onto the latest master. Can you help with finding out the cause?
https://github.com/pytorch/pytorch/pull/43752/checks?check_run_id=1052577205

**Comment by malfet** [comment]
> @walterddr @malfet Linter(clang-tidy) failed. I rebased onto the latest master. Can you help with finding out the cause?
> https://github.com/pytorch/pytorch/pull/43752/checks?check_run_id=1052577205


Please feel free to ignore this failure, as it has nothing to do with your change.

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**